File: 1ba/PRG.H2503/block01/aufgabe1.java
// Aufgabe 1: Rechtecksberechnung
// Presentation - Block 01

import java.util.Scanner;

public class aufgabe1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Geben Sie die Breite des Rechtecks ein: ");
        double breite = scanner.nextDouble();
        
        System.out.print("Geben Sie die Höhe des Rechtecks ein: ");
        double hoehe = scanner.nextDouble();
        
        double flaeche = breite * hoehe;
        
        double umfang = 2 * (breite + hoehe);
        
        System.out.println("\nErgebnisse für das Rechteck:");
        System.out.println("Fläche: " + flaeche);
        System.out.println("Umfang: " + umfang);
        
        scanner.close();
    }
}

File: 1.BA/PRG.H2503/golang/block01/aufgabe1.go
package main

import "fmt"

func main() {
	var breite float64
	var hoehe float64

	fmt.Print("Geben Sie die Breite des Rechtecks ein: ")
	fmt.Scanln(&breite)
	fmt.Print("Geben Sie die Höhe des Rechtecks ein: ")
	fmt.Scanln(&hoehe)

	flaeche := breite * hoehe
	umfang := 2 * (breite + hoehe)

	fmt.Println("\nErgebnisse für das Rechteck:")
	fmt.Println("Fläche: ", flaeche)
	fmt.Println("Umfang: ", umfang)
}

